krux_boto/cli.py

```diff
@@ -31,6 +32,8 @@ def __init__(self, name=NAME):
 
         self.boto = get_boto(self.args, self.logger, self.stats)
 
+        self.boto3 = get_boto3(self.args, self.logger, self.stats)
+
```

Comment:
So we always set up a boto *and* a boto3 object? I'm wondering if it would be better to select which one is needed via an arg to __init__() ... at least that way we can always refer to the object we're using as self.boto, which will make it easier to migrate if/when we want to make everything everywhere boto3-compatible?

Unless there are cases where we want to use the 2 modules in a single script, in which case ignore me :)